inductiva/utils/format_utils.py

```diff
@@ -40,55 +40,60 @@ def seconds_formatter(secs: float) -> str:
     return str(datetime.timedelta(seconds=round(secs)))
 
 
-def get_tabular_str(
-    rows,
-    columns,
-    default_col_space=15,
-    override_col_space=None,
-    formatters=None,
-) -> str:
-    """Converts a list of lists to a string table.
+def apply_formatters(table_data: dict, formatters: dict):
+    """Apply formatters to a list of lists of data (rows).
 
-    Temporary solution to display tables. `pandas` dependency to print tables
-    is overkill, but since we have `pandas` anyway for now, we can use it.
+    Args:
+        table_data : Dictionary of column names and lists of data.
+        formatters : Dictionary of column names and functions to
+            apply to that column's data.
```

Comment:
update docstring to conform with arg names.

Reply:
Done